e2e/src/recurringMeetings.spec.ts

```diff
@@ -404,6 +441,69 @@ test.describe('Recurring Meetings', () => {
     await expect(
       aliceMeetingsWidgetPage.getMeeting('My Meeting').card,
     ).toHaveCount(4);
+
+    await aliceElementWebPage.switchToRoom('My Meeting');
+
+    await expect(
+      aliceElementWebPage.locateChatMessageInRoom(
+        /A single meeting from a meeting series on October 3, 2040, 10:30 – 11:30 AM GMT\+2 is deleted/,
+      ),
+    ).toBeVisible();
+  });
+
+  test('should edit and delete a single recurring meeting', async ({
```

Comment:
I'm unsure the purpose of this test since it is basically the same as "should delete a single recurring meeting". Why is the edit important?  And isn't this something the unit tests already tested?

Reply:
Yes, it is better to have this in unit test, somehow was missing. I have added unit tests, thanks for the comment.